builtin/game/misc.lua

```diff
@@ -114,6 +114,10 @@ function core.get_connected_players()
 	return temp_table
 end
 
+function minetest.player_exists(name)
+	return minetest.get_auth_handler().get_auth(name) ~= nil
```

Comment:
i mean, if there is similar function in other builting parts, replace it with a call to this function